test-requirements.txt

```diff
@@ -1,8 +1,8 @@
 pytest==3.0.6
 mock
--e git+https://github.com/uc-cdis/cdisutils-test.git@0.2.3#egg=cdisutilstest
+-e git+https://github.com/uc-cdis/cdisutils-test.git@feat/rbac#egg=cdisutilstest
 -e git+https://github.com/uc-cdis/cdis-python-utils.git@0.1.7#egg=cdispyutils
--e git+https://github.com/uc-cdis/indexd.git@1.0.6#egg=indexd
+-e git+https://github.com/uc-cdis/indexd.git@ddc6bfd1e0cd4bbf27bc4ab7a7e00078b4d5c98c#egg=indexd
```

Comment:
update these?

Reply:
@paulineribeyre probably not indexd, but I'll update cdisutils-test for sure. Thanks!